doc/tutorials/starship.rst

```diff
@@ -37,46 +40,56 @@ To test connectivity, you can use the following command:
    You have successfully authenticated to the platform!
 
 
+.. _first_program:
+
 Executing your first program
 ============================
-The easiest way to execute a program using StarshipEngine is to create a blackbird script (an ``xbb`` file) and place it in your current working directory. Check the :ref:`blackbird tutorial <tutorial>` for how to create this file.
 
-For this example, we will use the following file and save it to ``test.xbb`` in our current working directory:
+The easiest way to execute a program using StarshipEngine is to create a Blackbird script (an ``xbb`` file)
+and place it in your current working directory. Check the :ref:`Blackbird tutorial <blackbird>` for how to create this file.
+
+For this example, consider the following Blackbird script, which represents a quantum program that matches
+exactly the gate layout of the Chip0 photonic hardware device. We will save the following file as ``test.xbb``
+in our current working directory:
 
 .. code-block:: python
 
-   name template_2x2_chip0
-   version 1.0
-   target chip0 (shots = 50)
+   name template_2x2_chip0     # Name of the program
+   version 1.0                 # Blackbird version number
+   target chip0 (shots = 50)   # This program will run on chip0 for 50 shots
 
-   float sqz0 = 1.0
-   float sqz1 = 1.0
```

Comment:
This gives the impression that the on-chip squeezing is a parametrized value, that anyone can modify; however, currently the squeezing value is fixed to 1.0. I moved it directly into the S2gate to avoid bugs due to people tempted to change the squeezing value :)

Of course, it won't fully stop anyone from changing the squeezing magnitude, just makes it a little less visible.

Reply:
Yep makes sense!